CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Push lint and breaking configuration to the registry.
```

Comment:
We may want to position this as a bug fix -- "Fixed bug to resume pushing lint and breaking configuration to the registry" or something along those lines.